.github/workflows/main.yml

```
@@ -49,7 +47,9 @@ jobs:
       fail-fast: false
       matrix:
         php-version:
-          - 7.1
+          - 7.4
+          - 8.0
+          - 8.1
```

IS there any reason to run CS on multiple PHP version when we run it with `--runtime-set php_version 70400` anyways?

```suggestion
          - 7.4
```

---

The codebase can only be "checked" against one runtime (is it beautified against php 7.4 or against php 8.1?). It makes sense to me to use the lowest one. If it would not be locked to a particular runtime, and there were fixes available exclusively for the 8.1 runtime for example, then the pipeline would fail, even though it should not, since different results are to be expected at different runtimes. I agree that the test is imperfect, but I see no easy solution.

---

Makes sense. At the very least it validates that our ruleset is consistent in behavior and can be executed on any of these PHP versions.